ui/collapsible-panel.reel/collapsible-panel.css

```diff
@@ -0,0 +1,22 @@
+/* Header */
+.montage-CollapsiblePanel > :nth-child(1) {
+    cursor: pointer;
+}
+
+.montage-CollapsiblePanel-bodyWrapper {
+    overflow: hidden;
+    max-height: 0;
+}
+
+.montage-CollapsiblePanel--transition {
+    -webkit-transition: max-height 0.25s ease;
+    -moz-transition: max-height 0.25s ease;
+    -o-transition: max-height 0.25s ease;
+    transition: max-height 0.25s ease;
+}
+
+.montage-CollapsiblePanel--active {
+    height: auto;
+    /* A popular way to animation auto height, we have to set a max-height to trigger the animation */
+    max-height: 1000px;
+}
```

Comment:
lines 12, 13, & 14: Are the -webkit, -moz, and -o prefixes necessary? http://caniuse.com/#feat=css-transitions suggests no current version of Chrome, Safari, Firefox, or Opera requires these, and we're not supporting legacy versions of those browsers.